Microsoft.Azure.CosmosRepository/src/Microsoft.Azure.CosmosRepository.csproj

```diff
@@ -45,7 +45,7 @@
     <PackageReference Include="Microsoft.Azure.Cosmos" Version="3.14.0" />
     <PackageReference Include="Microsoft.Extensions.Http" Version="5.0.0" />
     <PackageReference Include="Microsoft.Extensions.Logging" Version="5.0.0" />
```

Comment:
Now I'm curious, should these two be bumped down too? Is there a `3.1.10` version for them?